src/components/graph/graph.ts

```diff
@@ -314,10 +315,26 @@ export default class GraphComponent {
 
   // @ts-ignore // no unused variable
   private _enableClickToCenter(): void {
+    let el: any = null
+    if (document.getElementsByTagName('temp').length === 0) {
+      el = document.createElement('temp')
+      el.setAttribute('id', 'editor-container')
+      document.body.appendChild(el)
+    }
+
     this._nodes.on('click.centerOnNode', (d: any) => {
+      el = document.getElementById('editor')
+      let width: number = this._width
+
+      if (el === null) {
+        el = document.getElementsByTagName('temp')[0]
+        el = getComputedStyle(el).getPropertyValue('flex').split(' ')
+        width = this._width - parseInt(el[el.length - 1], 10)
+      }
+
       const {translation, scale} = this._getGraphTranslationAndScale()
       const {position} = this._graphToSVGPosition(d)
-      const x = translation[0] + this._width / 2 - position[0]
+      const x = translation[0] + width / 2 - position[0]
```

Comment:
We need to figure out a better way of doing this. We can't be relying on anything except what's passed in to this component (class). If we start assuming things about the rest of the application, then we're creating confusing code that doesn't respect good design principles (you know separation of concern and cohesion etc.)